packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { Div100vhContainer, Icon, MobileDrawer, ToggleSwitch, Text } from '@deriv/components';
 import { getPlatformSettings, routes, PlatformContext } from '@deriv/shared';
 import { localize, getAllowedLanguages, getLanguage } from '@deriv/translations';
-import { NetworkStatus } from 'App/Components/Layout/Footer';
+import NetworkStatus from 'App/Components/Layout/Footer';
```

Comment:
As `NetworkStatus` is not default export in the `Footer`, I think it should be `{ NetworkStatus }`.

Reply:
@hamid-deriv check [this](https://github.com/binary-com/deriv-app/pull/5902/files#r922665449) please